### PR TITLE
tests: fix failures with Pygments 2.1

### PR DESCRIPTION
**Pull request by nckx**

### CODE REVIEW

**Comment by justinmayer** [comment]
Looks good to me. @avaris / @ingwinlu?

**Comment by ingwinlu** [comment]
some ws changes that are not related to pygments, and the obvious 'do not submit pr's from master'

**Comment by nckx** [comment]
Nothing is unrelated: the diffs are taken straight from the test results. But I can fudge them a bit to add newlines, no problem.

File: pelican/tests/output/custom/unbelievable.html [hunk 1]
```diff
@@ -43,9 +43,9 @@ <h1 class="entry-title">
         </abbr>
 
         <address class="vcard author">
-                By <a class="url fn" href="./author/alexis-metaireau.html">Alexis Métaireau</a>
+                By                         <a class="url fn" href="./author/alexis-metaireau.html">Alexis Métaireau</a>
```
Comment (ingwinlu) [inline pelican/tests/output/custom/unbelievable.html]: this is definitely not caused by the pygments update. it was introduced via some change to the used theme but the tests never got updated.

Comment (nckx) [inline pelican/tests/output/custom/unbelievable.html]: You're right. Would renaming the commit to something closer to `tests: fix`? I'm afraid I don't have time to identify the seperate causes/commits.

File: pelican/tests/output/basic/unbelievable.html [hunk 2]
```diff
@@ -38,7 +38,7 @@ <h1 class="entry-title">
                 Published: Fri 15 October 2010
         </abbr>
 
-<p>In <a href="/category/misc.html">misc</a>. </p>
+<p>In <a href="/category/misc.html">misc</a>.</p>
```
Comment (ingwinlu) [inline pelican/tests/output/basic/unbelievable.html]: not pygments

File: pelican/tests/output/basic/unbelievable.html [hunk 3]
```diff
@@ -97,4 +97,4 @@ <h1 class="entry-title">
         </footer><!-- /#contentinfo -->
 
 </body>
-</html>
+</html>
```
Comment (ingwinlu) [inline pelican/tests/output/basic/unbelievable.html]: not pygments

File: pelican/tests/output/custom/unbelievable.html (hunk 1 above)
Comment (ingwinlu) [inline pelican/tests/output/custom/unbelievable.html]: not pygments

File: pelican/tests/output/custom/unbelievable.html [hunk 4]
```diff
@@ -43,9 +43,9 @@ <h1 class="entry-title">
...
         </address>
-<p>In <a href="./category/misc.html">misc</a>. </p>
+<p>In <a href="./category/misc.html">misc</a>.</p>
```
Comment (ingwinlu) [inline pelican/tests/output/custom/unbelievable.html]: not pygments

File: pelican/tests/output/custom_locale/posts/2010/octobre/15/unbelievable/index.html [hunk 5]
```diff
@@ -45,7 +45,7 @@ <h1 class="entry-title">
         <address class="vcard author">
                 By                         <a class="url fn" href="../../../../../author/alexis-metaireau.html">Alexis Métaireau</a>
         </address>
-<p>In <a href="../../../../../category/misc.html">misc</a>. </p>
+<p>In <a href="../../../../../category/misc.html">misc</a>.</p>
```
Comment (ingwinlu) [inline pelican/tests/output/custom_locale/posts/2010/octobre/15/unbelievable/index.html]: not pygments

**Comment by ingwinlu** [comment]
hey @nckx ; I tagged the lines that are not from pygments.
Can you put those into a separate commit from the other ones?

Then we are golden!

Also sorry for the long wait; rather busy with other stuff.

**Comment by nckx** [comment]
No problem, so am I. Thank you for the help, it would have taken me a day to figure out what is probably obvious to people who know these packages.

I'll fix up and resubmit.

**Comment by nckx** [comment]
Done, I think/hope.

**Comment by ingwinlu** [comment]
thank you @nckx 

@justinmayer lgtm

**Comment by justinmayer** [comment]
Many thanks to Tobias for the fix and to Winlu for the review.

**Pull request merged**